store/store.go

```diff
@@ -467,6 +467,9 @@ type requestOptions struct {
 	Data        []byte
 }
 
+// Store method for doing requests to store server
+var doStoreRequest = (*Store).doRequest
```

Comment:
propably easiest is to check for the device auth,  without involving a user (which ends up I think needing to fake also the purchases endpoint)